format/avro/testdata/twitter.fqtest

```diff
@@ -0,0 +1,50 @@
+$ fq verbose twitter.avro
```

Comment:
I've usually kept some comment where source test files come from or how to create them. To make it easier in the future to debug or add things